libsel4/sel4_arch_include/aarch64/interfaces/sel4arch.xml

```diff
@@ -106,10 +106,10 @@
                     Map an upper page directory
```

Comment:
In 40 bit PA configurations, should this PUD object be not available in the API?

Reply:
Good point. Currently the kernel will `fail` if an invalid object is passed to `decodeARMMMUInvocation`.

Reply:
added a commit to resolve this